src/main/resources/migration/common/V7__adjudication_id.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE additional_days_awarded ALTER COLUMN adjudication_id TYPE INTEGER USING adjudication_id::integer;
```

Comment:
Is Integer in the db and codebase big enough for adjudication numbers? Should it be a long/bigint?

Reply:
good spot @ldlharper , it's big enough when comparing to the actual data - however, the Adjudications service uses Long... so we should do the same to keep it consistent. Will modify